src/net/sourceforge/kolmafia/swingui/panel/RestoreOptionsPanel.java

```diff
@@ -148,7 +148,7 @@ private String getSettingString(final JCheckBox[] restoreCheckbox) {
     return restoreSetting.toString();
   }
 
-  private float getPercentage(final JComboBox option) {
+  private float getPercentage(final JComboBox<?> option) {
```

Comment:
Similarly, can this be replaced with a JComboBox<String>? (ditto below in setSelectedIndex)

Reply:
sure, I guess, since they are private methods... not like someone is gonna pop in with a JComboBox of a different generic and will be sad they can't use that method...